pkgs/top-level/python-packages.nix

```diff
@@ -88,6 +123,9 @@ in {
 
   vowpalwabbit = callPackage ../development/python-modules/vowpalwabbit { pythonPackages = self; };
 
+#   unique = assertUniqueNames [ self.django_1_6.name self.django_1_8.name self.numpy.name ];
+  unique = assertUniquePythonPackages [ self.django_1_6 self.django_1_8 self.numpy ];
+
```

Comment:
@garbas this is what you meant, right? This we could now implement in the builder.

Reply:
@FRidh 👍 looks like a good start

I suppose this should be done automatically by buildPythonPackage

and sorry for the delay .. i'm slowly getting back to online life 😄